core/state_transition.go

```diff
@@ -563,8 +613,21 @@ func (st *StateTransition) distributeTxFees(eHardfork bool) error {
 		gatewayFeeRecipient = &common.ZeroAddress
 	}
 
-	caller := &vmcontext.SharedEVMRunner{EVM: st.evm}
-	governanceAddress, err := contracts.GetRegisteredAddress(caller, params.GovernanceRegistryId)
+	var (
+		governanceAddress common.Address
+		err               error
+	)
+	if !st.evm.ChainConfig().Faker {
```

Comment:
Please remove the `Faker` check. It seems like it's primarily for the 1559 transition test (so it's actually not testing the full codepath in the test).